src/default-retrieve-file-handler.ts

```diff
@@ -1,23 +0,0 @@
-import * as fs from 'fs';
-import { transpileIfTypescript } from './transpile-if-ts';
-
-export function defaultRetrieveFileHandler(path) {
-  // Trim the path to make sure there is no extra whitespace.
-  path = path.trim();
-
-  // This was removed because it seems that we can't use cache while expecting correct results
-  // TODO: check correctness and performance with file caching 
-  // if (path in fileContentsCache) {
-  //   return fileContentsCache[path];
-  // }
-
-  var contents: string;
-  try {
-    contents = fs.readFileSync(path, 'utf8');
-    contents = transpileIfTypescript(path, contents);
```

Comment:
I have no problem removing the repeat usage if it isn't needed. It's just that I remember having tried that and it not working (the tests passed because Jest got the transpiled code but the mapped line numbers were wrong because `source-map-support` didn't get the sourcemaps).